subcontracted_service/tests/test_subcontracted_service.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+# Author: Damien Crier
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp.tests.common import TransactionCase
+
+
+class TestSubcontractedService(TransactionCase):
+    def setUp(self):
+        super(TestSubcontractedService, self).setUp()
+        # 1. find a supplier
+        self.supplier = self.env.ref('base.res_partner_1')
+
+        # 2. create a service product unconfigured
+        values = {'name': 'Service Subcontracted',
+                  'type': 'service',
+                  'seller_ids': [(0, 0, {
+                      'name': self.supplier.id,
+                      'price': 100.0,
+                  })]
+                  }
+        self.pdt_service = self.env['product.product'].create(values)
+        # 3. create procurement rules in companies
+        companies = self.env['res.company'].search([])
+        companies._set_subcontracting_service_proc_rule()
+
+        # 4. find a customer
+        self.customer = self.env['res.partner'].search(
+            [('customer', '=', True)],
+            limit=1
+        )
+
+    def test_01_product_no_configured(self):
+        # create a SO
```

Comment:
There is no real dependency on the sale module, you can just test the creation of a procurement of a service product. This way the module would have less constrains to be used in different scenarios. For instance, to subcontract services in manufacturing orders (our case), which has nothing to do with sales.